packages/hash/api/src/graphql/resolvers/knowledge/page/page.ts

```diff
@@ -20,3 +25,53 @@ export const knowledgePage: ResolverFn<
 
   return mapPageModelToGQL(pageModel);
 };
+
+export const createKnowledgePage: ResolverFn<
+  Promise<UnresolvedKnowledgePageGQL>,
+  {},
+  LoggedInGraphQLContext,
+  MutationCreateKnowledgePageArgs
+> = async (
+  _,
+  { ownedById, properties: { title, prevIndex } },
+  { dataSources: { graphApi } },
+) => {
+  const page = await PageModel.createPage(graphApi, {
```

Comment:
it's barely used so this is a _**nit**_ but for consistency should we call this `pageModel`?

Reply:
Addressed in https://github.com/hashintel/hash/pull/1157/commits/d7b10899672eb280db9546bdc7ff253faf8d74e5